pkg/actions/install.go

```diff
@@ -32,11 +32,10 @@ func InstallCommand(c *cli.Context) {
 	tag := c.String("tag")
 	jsonOutput := c.Bool("json") || c.GlobalBool("json")
 
-	imageArr := [2]string{"docker.io/eclipse/codewind-pfe-amd64:",
-		"docker.io/eclipse/codewind-performance-amd64:"}
-
-	targetArr := [2]string{"codewind-pfe-amd64:",
-		"codewind-performance-amd64:"}
+	imageArr := [2]string{
+		"docker.io/eclipse/codewind-pfe-amd64:",
+		"docker.io/eclipse/codewind-performance-amd64:",
+	}
```

Comment:
Could we define this as:
```
	imageArr := [2]string{
		"docker.io/eclipse/codewind-pfe-amd64:" + tag,
		"docker.io/eclipse/codewind-performance-amd64:" + tag,
	}
```
I don't think we can use the names without the tag ever as they end in `:` so we can get away with just appending tag once and reduce the scope for forgetting.